src/docs/development/data-and-backend/state-mgmt/simple.md

```diff
@@ -0,0 +1,327 @@
+
+The only required argument of the `ScopedModelDescendant` widget is the builder. Builder is a function that will be called whenever the model changes. (In other words, when you call `notifyListeners()` in your model, all the builder methods of all the corresponding `ScopedModelDescendant` widgets are called.)
+
+The builder is called with three attributes. The first one is `context`, which you also get in every build method. 
+
+The second attribute is `child`, which is there for optimization. If you have a large widget subtree under your `ScopedModelDescendant` that _doesn't_ change when the model changes, you can construct it once and get it through the builder.
+
+```dart
+ScopedModelDescendant<CartModel>(
+  builder: (context, child, cart) => Column(
+       children: [
+         // Use SomeExpensiveWidget here, without rebuilding every time.
+         child,
+         Text("Total price: ${cart.totalPrice}"),
+       ],
+     ),
+  // Build the expensive widget here.
+  child: SomeExpensiveWidget(),
+);
+```
+
+The third argument of the builder function is the model. That's what we were asking for in the first place. You can use the data in the model to define what the UI should look like at any given point.
+
+It is best practice to put your `ScopedModelDescendant` widgets as deep in the tree as possible. You don't want to rebuild large portions of the UI just because some detail somewhere changed.
+
+```dart
+// DON'T DO THIS
+ScopedModelDescendant<CartModel>(
+  builder: (context, child, cart) {
+    return HumongousWidget(
+      // ...
+      child: AnotherMonstrousWidget(
+        // ...
+        child: Text('Total price: ${cart.totalPrice}'),
+      )
+    );
+  },
+);
+```
+
+Instead:
+
+```dart
+// DO THIS
+HumongousWidget(
+  // ...
+  child: AnotherMonstrousWidget(
+    // ...
+    child: ScopedModelDescendant<CartModel>(
+      builder: (context, child, cart) {
+        return Text('Total price: ${cart.totalPrice}');
+      },
+    ),
+  ),
+);
+```
+
+### ScopedModel.of
+
+Sometimes, you don't really need the _data_ in the model to change the UI but you still need to access it. For example, a `ClearCart` button wants to allow the user to remove everything from the cart. It doesn't need to display the contents of the cart, it just needs to call the `clear()` method.
+
+We could use `ScopedModelDescendant<CartModel>` for this, but that would be wasteful. We'd be asking the framework to rebuild a widget that doesn't need to be rebuilt. 
+
+For this use case, we can use `ScopedModel.of`. 
+
+```dart
+ScopedModel.of<CartModel>(context).add(item);
+```
+
+Using the above line in a build method will not lead to any rebuilding. 
```

Comment:
I think this line could be misinterpreted -- it will lead to the connected ScopedModelDescendants to rebuild (some, some rebuilding is happening!), but it will not rebuild the *current* Widget that is accessing the scoped model in this way.

Maybe something like: "Using the above line in a build method will not cause this Widget to rebuild when `notifyListeners` is called"

Reply:
ack